readthedocs/projects/forms.py

```diff
@@ -660,6 +660,14 @@ class DomainBaseForm(forms.ModelForm):
     class Meta:
         model = Domain
         exclude = ['machine', 'cname', 'count']  # pylint: disable=modelform-uses-exclude
+        error_messages = {
+            'domain':  {
+                'unique': (
+                    'This domain is already configured. '
+                    'Please choose another.'
+                ),
```

Comment:
Any suggestions for this message such as this goes well with same project as well as other projects?